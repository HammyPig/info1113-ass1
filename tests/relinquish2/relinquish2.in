relinquish 100000 111111
add member imhungry
rELINQUISh 100000 111111
add collection duplicates.csv
list available
relinquish 900000 111111
RELINQUISH 100000 999999
reLINquish 100000 111111
rent 100000 111111
list all long
list available
member books 100000
relinQUish 100000 111111
list all long
list available
member books 100000
member history 100000