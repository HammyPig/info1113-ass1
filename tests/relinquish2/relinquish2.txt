recognises when member(s) or book(s) don't exist, otherwise relinquish works as intended (cannot relinquish not rented book, and relinquish frees book and member connection)