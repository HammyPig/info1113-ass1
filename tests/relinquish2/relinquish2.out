user: No members in system.

user: Success.

user: No books in system.

user: 3 books successfully added.

user: To Kill a Mockingbird (Harper Lee)
To Kill a Mockingbird (Harper Lee)
The Iliad (Homer)

user: No such member in system.

user: No such book in system.

user: Unable to return book.

user: Success.

user: 111111: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Rented by: 100000.

111112: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

111113: The Iliad (Homer, Poetry)
Currently available.

user: To Kill a Mockingbird (Harper Lee)
The Iliad (Homer)

user: To Kill a Mockingbird (Harper Lee)

user: Success.

user: 111111: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

111112: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

111113: The Iliad (Homer, Poetry)
Currently available.

user: To Kill a Mockingbird (Harper Lee)
To Kill a Mockingbird (Harper Lee)
The Iliad (Homer)

user: Member not currently renting.

user: To Kill a Mockingbird (Harper Lee)

user: Ending Library process.
